File: Programming Assignment/Assignment 14/program4.c
#include<stdio.h>
void Display(int iRow,int iCol)
{

int i=0,j=0;

for(i=1;i<=iRow;i++)
{
	if(i>=2 && i<6)
	{
		for(j=1;j<=iCol;j++)
		 {
             if(j>=2 && j<6)
			 {
				 if(i==j)
				 
					 printf("*\t");
					 else
						 printf(" \t");
			}
			else
			{
				printf("*\t");
			}
		 }
	}
  	else
	{
		for(j=1;j<=iCol;j++)
		{
			printf("*\t");
		}
	}
	printf("\n");
}
}
		 












int main()
{
	int iValue1=0,iValue2=0;
	
	printf("Enter Number of row\n");
	scanf("%d",&iValue1);
	
	printf("Enter Number of column\n");
	scanf("%d",&iValue2);
	
	Display(iValue1,iValue2);
	
	return 0;
	
	
	
}

File: C# LB Assignment/Assignment 10/program4.cs
using System;

class Test
{
public void Display(int iRow,int iCol)
{
for(int i=iRow;i>=1;i--)
{
	for(int j=1;j<=iCol;j++)
	{
		if((j%2)==0)
		{
		Console.Write("#\t");
	     }
	     else
	     {
	     	Console.Write("*\t");
	     }
	}
	Console.WriteLine();

}
}
}

class pgm1
{

static void Main(string []arg)
{
int no1,no2;

Console.WriteLine("Enter no of row");
no1=Convert.ToInt32(Console.ReadLine());

Console.WriteLine("Enter no of col");
no2=Convert.ToInt32(Console.ReadLine());

Test obj=new Test();

obj.Display(no1,no2);

}

	}